### PR TITLE
CLOUD-513 Add license tests

### PR DESCRIPTION
**Pull request by hors**

### CODE REVIEW

**Comment by JNKPercona** [comment]
Check license is ok

**Comment by JNKPercona** [comment]
License check is ok.

File: Jenkinsfile [hunk 1]
```diff
@@ -77,69 +21,57 @@ pipeline {
...
-                            export IMAGE=\$DOCKER_TAG
-                            ./e2e-tests/build
-                            docker logout
-                        "
-                    '''
-                }
-                stash includes: 'results/docker/TAG', name: 'IMAGE'
-                archiveArtifacts 'results/docker/TAG'
+               sh """
+                   IS_GPL=\$(license-detector ${WORKSPACE} | grep GPL) || true
```
Comment (delgod) [inline Jenkinsfile]: lets store allowed the output of checked here and `diff` it here

**Review by delgod — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (hors) [reply Jenkinsfile]: done

**Comment by JNKPercona** [comment]
License check is ok.

**Review by delgod — APPROVED** [review]

**Pull request merged**